tests/integration_tests/security/spectre-mitigations

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
can we rename so it has a .sh extension?

I am thinking if we want to run shellcheck in the future that would be an easy way to collect all shell scripts: `find . -name "*.sh"`

Reply:
As we see with `devtool` a few other scripts throughout the repo the standard is to not use the `.sh` extension.